binr/r2pm/r2pm

```diff
@@ -327,16 +330,24 @@ R2PM_GIT() {
 	cd "${R2PM_GITDIR}"
 	if [ -d "${DIR}" ]; then
 		cd "${DIR}"
-		git checkout master
-		git pull
+		if [ "${R2PM_GITMASTER}" = "master" ]; then
+			git checkout master
+			git pull
+		else
+			git checkout ${R2PM_GITMASTER}
+		fi
 	else
 		git clone --depth 1 --recursive "${URL}" "${DIR}" || R2PM_FAIL "Oops"
 		cd "${DIR}" || R2PM_FAIL "Oops"
 	fi
-	r2 -qv | grep -q git || { # r2 >= 4.5
-		git checkout "r2-${R2_VERSION}"
-		git pull
-	}
+	if [ "${R2PM_GITMASTER}" = "master" ]; then
+		r2 -qv | grep -q git || { # r2 >= 4.5
+			git checkout "r2-${R2_VERSION}"
```

Comment:
theres another PR to fix this R2_VERSION thing. see https://github.com/radareorg/radare2/pull/17650/files

can you rename this var to R2VERSION instead?

Reply:
OK

Reply:
ping,